reason_4.0/lib/core/minisite_templates/modules/events.php

```diff
@@ -3573,17 +3573,22 @@ function show_list_export_links()
 		}
 		
 		$query_string = $this->construct_link(array('start_date'=>$start_date,'view'=>'','end_date'=>'','format'=>'ical'));
+		$webcal_url = 'webcal://'.REASON_HOST.$this->parent->pages->get_full_url( $this->page_id ).$query_string;
+		$gcal_url = 'https://calendar.google.com/calendar/render?cid='.str_replace(array('&amp;','='),array('%26','%3D'),$webcal_url);
```

Comment:
It would be more robust to use urlencode($webcal_url) instead of this targeted str_replace()